src/popup/components/TransferSendForm.vue

```diff
@@ -250,7 +252,7 @@ export default defineComponent({
       () => formModel.value.selectedAsset?.contractId !== AETERNITY_CONTRACT_ID,
     );
     const isMaxValue = computed<boolean>(
-      () => isNumbersEqual(formModel.value.amount, max.value),
+      () => isNumbersEqual(+formModel.value.amount, +max.value),
```

Comment:
```suggestion
      () => +formModel.value.amount === +max.value,
```
🤔

Reply:
Makes sense as those 2 are not BigNumbers. Changed...